src/index.tsx

```diff
@@ -206,7 +207,7 @@ function PrefabTestProvider({ config, children }: PropsWithChildren<TestProps>)
       loading: false,
       prefab: prefabClient,
       keys: Object.keys(config),
-      settings: {},
+      settings: { apiKey: apiKey ?? "fake-api-via-the-test-provider" },
```

Comment:
@semanticart super nitpicky, `key` is missing from the string, could change to `fake-api-key-via-the-test-provider`. But totally up to you.

Reply:
I like that! Thank you